src/util/tile-checker.js

```diff
@@ -0,0 +1,27 @@
+var $ = require('jquery');
+var getValue = require('./get-object-value');
+var limitsSVG = require('../../themes/img/timeout.svg');
+
+var buildErrorTile = function (svg) {
+  return 'data:image/svg+xml;base64,' + window.btoa(svg);
```

Comment:
IE 10 according to [MDN](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/btoa), but yep, I don't trust IE, so a manual check would be nice.